daq/report.py

```diff
@@ -82,16 +85,22 @@ def finalize(self):
             LOGGER.info('Copying report to %s', self._alt_path)
             shutil.copyfile(self.path, self._alt_path)
 
+    def _write_table(self, items):
+        self._writeln(self._TABLE_MARK + self._TABLE_MARK.join(items) + self._TABLE_MARK)
+
     def _write_test_summary(self):
         self._writeln(self._TEST_SEPARATOR % self._SUMMARY_LINE)
-        self._writeln(self._PRE_START_MARKER)
```

Comment:
Considering how the test summary is written, would it impact the way _append_file(called in _copy_test_reports) copies the report? To be mroe specific, do we need to make this change in _append_file as well?

Reply:
No -- the difference is that *this* function is converted now to write a markdown table in-line, so no longer needs the PRE marking since it's using native markdown (where as before it was just text).  The other use is still copying an external file in, so still needs the pre marking around it.